tests/E2E/test/common.webdriverio.js

```diff
@@ -179,10 +179,12 @@ module.exports = {
     if (typeof global.selenium_host !== 'undefined') {
       options.host = global.selenium_host;
     }
+    else global.selenium_host = 'localhost';
```

Comment:
Please :smile_cat: 

```
if (typeof global.selenium_host !== 'undefined') {
      options.host = global.selenium_host;
    } else {
      // We need to set it to localhost for "got" library
      global.selenium_host = 'localhost';
    }
```

PS: Same for the port

Reply:
Thanks for suggestion
done